tools/dataProtector/dataProtectorCore/processProtectedData.md

```diff
@@ -210,6 +210,45 @@ const taskid = await dataProtectorCore.processProtectedData({
 });
 ```
 
+### onStatusUpdate
+
+`OnStatusUpdateFn<ProcessProtectedDataStatuses> | undefined`
+
+Callback function to be notified at intermediate steps.
+
+<!-- prettier-ignore-start -->
+```ts twoslash
+import { IExecDataProtectorCore, getWeb3Provider } from '@iexec/dataprotector';
+
+const web3Provider = getWeb3Provider('PRIVATE_KEY');
+const dataProtectorCore = new IExecDataProtectorCore(web3Provider);
+// ---cut---
+
+const taskid = await dataProtectorCore.processProtectedData({
```

Comment:
Wrong returned variable. (It returns an object)

```
const { taskId } = await dataProtectorCore.processProtectedData({
```

or

```
const processProtectedDataResponse = await dataProtectorCore.processProtectedData({
```

Reply:
yeah my bad, a copy paste error